agent/src/beerocks/slave/son_slave_thread.h

```diff
@@ -200,6 +200,7 @@ class slave_thread : public beerocks::socket_thread {
     sSlaveBackhaulParams backhaul_params;
     beerocks_message::sNodeHostap hostap_params;
     beerocks_message::sApChannelSwitch hostap_cs_params;
+    std::vector<beerocks::message::sWifiChannel> hardware_supported_channels;
```

Comment:
Apparently the message got lost somewhere: there is no such thing as hardware supported channels. There are supported channels, and there are channel preferences. What needs to be added is channel preferences, and ACS needs to use that instead of supported_channels.

Reply:
Changed the previously  called `supported_channels` to `preferred_channels` throughout the codebase